docs/source/numpy.rst

```diff
@@ -130,6 +130,8 @@ Slicing and indexing
 +=====================================================+=====================================================+
 | ``a[3, 2]``                                         | ``a(3, 2)``                                         |
 +-----------------------------------------------------+-----------------------------------------------------+
+| ``a.flat[4]``                                       | ``a[4]``                                            |
```

Comment:
We could also add `a(4)` on a new line for C++14? Or do you find it too much confusing?

Reply:
I didn't know the forwarding to `operator()` ... Always learning !
I just pushed a new rebased commit.